content/md/what-is/what-about-reference-entities.md

```diff
@@ -0,0 +1,39 @@
+---
+id: what-about-reference-entities
+themes: first-steps
+title: What about **reference entities**?
+popular: true
+ee-only: true
+related:
```

Comment:
Don't forget to add your related articles at the end. ;)

Reply:
yep